### PR TITLE
Icon support for Button and FlatLink usage

### PR DESCRIPTION
**Pull request by hungqt**
* Icon wrapper for rightIcon and leftIcon to be used with Button and FlatLink.
* Closing #15 

Example usage: 
```
Button.primary
    |> Button.view [] [ text "Click me", Icons.rightIcon Icons.info ]
```

Screenshot:
image

NB: Button design is the old one, will be updated with the new after #13 is merged

### CODE REVIEW

**Review by fredrikloberg — APPROVED** [review]
Looks good 💯

**Comment by fshauge** [comment]
#13 has been merged

**Comment by fshauge** [comment]
Broken preview? Secondary and tertiary buttons are also disabled 😅

**Comment by hungqt** [comment]
> Broken preview? Secondary and tertiary buttons are also disabled 😅

Oh, more like broken code 💀  my bad, brb fix

File: explorer/src/Stories/Button.elm [hunk 1]
```diff
@@ -13,26 +16,50 @@ stories =
...
-                Button.tertiary
-                    |> Button.view [] [ text "Click me" ]
+                Html.div [ css [ children [ everything [ marginBottom (rem 1) ] ] ] ]
+                    [ Button.tertiary
+                        |> Button.view [ disabled True ] [ text "Click me" ]
+                    , Button.tertiary
+                        |> Button.view [ disabled True ] [ text "Click me", Icons.rightIcon Icons.info ]
+                    , Button.tertiary
+                        |> Button.view [ disabled True ] [ Icons.leftIcon Icons.info, text "Click me" ]
+                    ]
```
Comment (fshauge) [inline explorer/src/Stories/Button.elm:62]: These shouldn't be disabled

Comment (hungqt) [reply explorer/src/Stories/Button.elm]: Fixed 🤣

**Review by fshauge — APPROVED** [review]
Looks good 🤩

**Pull request merged**